main.go

```diff
@@ -381,18 +430,57 @@ func runCLIApp(c *cli.Context) (err error) {
 	{
 		mfs, err = mountWithArgs(bucketName, mountPoint, flags, mountConfig)
 
-		if err == nil {
+		// This utility is to absorb the error
+		// returned by daemonize.SignalOutcome calls by simply
+		// logging them as error logs.
+		callDaemonizeSignalOutcome := func(err error) {
+			if err2 := daemonize.SignalOutcome(err); err2 != nil {
+				logger.Errorf("Failed to signal to daemon: %v", err2)
```

Comment:
Error strings should not be capitalized: go/go-style/decisions#error-strings

Reply:
That's only for `fmt.Errorf` . The guideline says that the printed log message can be as needed.